forge-ai/src/main/java/forge/ai/AiBlockController.java

```diff
@@ -776,15 +772,44 @@ private void reinforceBlockersAgainstTrample(final Combat combat) {
                 continue;
```

Comment:
_Thorn Elemental_ ruling: 
> If blocked by a creature with banding, the defending player decides whether or not the damage is assigned “as though it weren’t blocked”. 

➡️ the part that checks for the static should be moved directly above the original block

Reply:
Hmm, not sure if I completely understand how... Can you please clarify a little or help with this one?

Reply:
banding still trumps in presence of that static
so it wouldn't be smart to skip completely if you can make it as a banding block

Reply:
Alrighty, I tried something, but not sure if that's exactly how you wanted it :/

Reply:
Sure, I think I'll clean it up even more with a few other changes later